README.md

```diff
@@ -152,14 +152,14 @@ title: ...
 ...
 sidebar_custom_props:
   offerings:
-    - bdp
+    - cdi
 ...
 ---
 ```
 
 This will result in an icon appearing in the sidebar, as well as an automatic banner on the page, specifying that the docs only apply to a given offering.
 
-The available values are: `bdp` and `community`. Do not specify both values at once — if a piece of documentation is relevant to all offerings, there should be no `offerings` property as that’s the default.
+The available values are: `cdi` and `community`. Do not specify both values at once — if a piece of documentation is relevant to all offerings, there should be no `offerings` property as that’s the default.
```

Comment:
Should we start calling it “self-hosted” instead of “community”? I think self-hosted customers sometimes find it difficult to find the relevant materials.